src/modules/dayCycle/DayCycleMoment.ts

```diff
@@ -9,7 +9,21 @@ export default class DayCycleMoment {
 
     get tooltip(): string {
         const tooltip = [];
-        tooltip.push(this.description);
+        const dayCycleSchedule: Array<string> = [
+            'Dawn : 06:00 - 07:00',
+            'Day: 07:00 - 17:00',
+            'Dusk: 17:00 - 18:00',
+            'Night: 18:00 - 06:00',
+        ];
+
+        dayCycleSchedule.forEach((schedule, dayCyclePart) => {
+            if (dayCyclePart === this.type) {
+                tooltip[dayCyclePart] = (`<b>${schedule}</b>`);
```

Comment:
The bold doesn't stand out very well to me (maybe just because of dark theme?), could we get an icon or something in front of the current part as well?